src/doc/src/reference/environment-variables.md

```diff
@@ -36,16 +36,18 @@ system:
   invocations that Cargo performs. In contrast with [`cargo rustdoc`], this is
   useful for passing a flag to *all* `rustdoc` instances. See
   [`build.rustdocflags`] for some more ways to set flags. This string is
-  split by whitespace; for a more robust encoding of multiple arguments,
-  set `CARGO_ENCODED_RUSTDOCFLAGS` instead with arguments separated by
-  `0x1f` (ASCII Unit Separator).
+  split by whitespace; for a more robust encoding of multiple arguments, 
+  see `CARGO_ENCODED_RUSTDOCFLAGS`.
+* `CARGO_ENCODED_RUSTDOCFLAGS` -  A list of custom flags separated by `0x1f` 
+  (ASCII Unit Separator) to pass to all `rustdoc` invocations that Cargo performs.
```

Comment:
I'm mixed on the fact that the old variables have some context not documented in the new variables but this is enough of an improvement I don't think its worth blocking this for the discussion and any updates.